java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -568,7 +568,8 @@ private static Object[] serverManagementCommands(BaseTransaction<?> transaction)
                 .configResetStat()
                 .lolwut(1)
                 .flushall()
-                .flushall(ASYNC);
+                .flushall(ASYNC)
+                .dbsize();
```

Comment:
Hopefully this isn't flaky.

Reply:
I don't think it will be flaky because we do a synchronous flushall before the asynchronous flushall. The database should already be empty before and after the second flushall